test/unit/math/prim/mat/fun/scale_matrix_exp_multiply_test.cpp

```diff
@@ -30,6 +30,12 @@ inline void test_scale_matrix_exp_multiply() {
 }
 
 TEST(MathMatrix, scale_matrix_exp_multiply) {
+  // the helper above doesn't handle 0 size inputs
+  const double t = 1.0;
+  Eigen::MatrixXd A(0, 0);
+  Eigen::MatrixXd B(0, 0);
```

Comment:
Can you add a test for A(0,1) and B(1,2)?

Reply:
Added.